cpp/modules/deck.gl/layers/test/line-layer-test.cpp

```diff
@@ -71,8 +71,8 @@ class LineLayerTest : public ::testing::Test {
     EXPECT_TRUE(widthBuilder.Finish(&widths).ok());
 
     std::vector<std::shared_ptr<arrow::Field>> schemaVector = {
```

Comment:
Do we need `schemaVector`? Can we just use initializer syntax in `Schema`?

Reply:
That kind of type inferring does not seem to work well with `std::make_shared`, as it does in the actual constructors, so that does not compile